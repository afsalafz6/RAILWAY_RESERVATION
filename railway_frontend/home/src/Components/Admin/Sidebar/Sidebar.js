import React, { Component } from "react";

class Sidebar extends Component {
  render() {
    return (
      <div>
        <ul id="slide-out" className="sidenav">
          <li>
            <div className="user-view">
              <div className="background">
                <img src="images/sidebar.jpg" alt="ssss" />
              </div>
              <a href="#user">
                <img className="circle" src="images/profile.jpg" alt="ddd" />
              </a>
              <a href="#name">
                <span className="white-text name">Admin</span>
              </a>
              <a href="#email">
                <span className="white-text email">admin@rail.com</span>
              </a>
            </div>
          </li>

          <li>
            <div className="divider"></div>
          </li>
          <li>
            <a href="#!" className="subheader">
              Menu
            </a>
          </li>
          <li>
            <a className="waves-effect" href="/">
              Home
            </a>
            <a className="waves-effect" href="/dashboard">
              Dashboard
            </a>
            <a className="waves-effect" href="/traindetails">
              Train Details
            </a>
            <a className="waves-effect" href="/userdetails">
              User Details
            </a>
            <a className="waves-effect" href="/allbooking">
              Booking Details
            </a>
          </li>
        </ul>
        {/* <a href="/sidebar" data-target="slide-out" className="sidenav-trigger">
          <i className="material-icons">menu</i>
        </a> */}
      </div>
    );
  }
}

export default Sidebar;
