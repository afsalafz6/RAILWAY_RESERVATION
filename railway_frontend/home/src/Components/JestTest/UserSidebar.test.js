import React from 'react';
import Sidebar from '../User/UserSidebar/UserSidebar.js'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Sidebar />)
    .toJSON();
    console.log(tree)
    expect(tree).toMatchSnapshot();
 });