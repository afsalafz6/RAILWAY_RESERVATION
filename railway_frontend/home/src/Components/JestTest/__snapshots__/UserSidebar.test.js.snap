// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <ul
    className="sidenav"
    id="slide-out"
  >
    <li>
      <div
        className="user-view"
      >
        <div
          className="background"
        >
          <img
            alt="ssss"
            src="images/sidebar.jpg"
          />
        </div>
        <a
          href="/profile"
        >
          <img
            alt="profile"
            className="circle"
            src="images/profile.jpg"
          />
        </a>
        <a
          href="#name"
        >
          <span
            className="white-text name"
          >
            
          </span>
        </a>
        <a
          href="#email"
        >
          <span
            className="white-text email"
          >
            
          </span>
        </a>
      </div>
    </li>
    <li>
      <div
        className="divider"
      />
    </li>
    <li>
      <a
        className="subheader"
        href="#!"
      >
        Menu
      </a>
    </li>
    <li>
      <a
        className="waves-effect"
        href="/"
      >
        Home
      </a>
      <a
        className="waves-effect"
        href="/usrdash"
      >
        Dashboard
      </a>
      <a
        className="waves-effect"
        href="/profile"
      >
        Profile
      </a>
      <a
        className="waves-effect"
        href="/alltrain"
      >
        All Train
      </a>
      <a
        className="waves-effect"
        href="/bookings"
      >
        Bookings
      </a>
    </li>
  </ul>
</div>
`;
